ci/matrix.xml

```diff
@@ -277,6 +278,30 @@
         'set scalaJSUseRhino in Global := true' test
   ]]></task>
 
+  <task id="sbtplugin-test-sbt1"><![CDATA[
```

Comment:
Mostly the `^^` thingy, which I would like to avoid at all in the 0.13.x variant. Also the `echo` is annoying, because if we do it in the 0.13.x job too, then nothing is testing what's stored *in the file* `sbt-plugin-test/project/build.properties` in the repo.

Given my level in bash-fu, I found it easier to copy-paste. But if you have a concrete suggestion for the two above issues, that'd be nice.